pennylane/ops/qubit.py

```diff
@@ -2159,6 +2159,16 @@ def _matrix(cls, *params):
 
         return U
 
+    @staticmethod
+    def decomposition(U, wires):
+        # Decompose arbitrary single-qubit unitaries as the form RZ RY RZ
+        if qml.math.shape(U) == (2, 2):
+            wire = Wires(wires)[0]
+            decomp_ops = qml.transforms.decompositions.zyz_decomposition(U, wire)
+            return decomp_ops
+
+        raise NotImplementedError("Decompositions only supported for single-qubit unitaries")
```

Comment:
Not sure why this isn't getting caught; the corresponding test is on line 1593 of the `test_qubit_ops` file.

Reply:
Ah, I think this is a well-known coverage bug. Fine to ignore.